pkg/generator/appconfiguration/generator/workload/service_generator.go

```diff
@@ -76,13 +76,28 @@ func (g *workloadServiceGenerator) Generate(spec *models.Spec) error {
 		spec.Resources = make(models.Resources, 0)
 	}
 
+	uniqueAppName := appconfiguration.UniqueAppName(g.project.Name, g.stack.Name, g.appName)
+
 	// Create a slice of containers based on the app's
-	// containers.
-	containers, err := toOrderedContainers(service.Containers)
+	// containers along with related volumes and configMaps.
+	containers, volumes, configMaps, err := toOrderedContainers(service.Containers, uniqueAppName)
 	if err != nil {
 		return err
 	}
 
+	// Create ConfigMap objects based on the app's configuration.
+	for _, cm := range configMaps {
+		cm.Namespace = g.project.Name
+		if err = appconfiguration.AppendToSpec(
+			models.Kubernetes,
+			appconfiguration.KubernetesResourceID(cm.TypeMeta, cm.ObjectMeta),
+			spec,
+			&cm,
```

Comment:
do not use reference to loop iterator variable. 
https://hackernoon.com/dont-make-these-5-golang-mistakes-3l3x3wcw

Reply:
Oops..., sorry for not noticing this mistake.